pkg/apis/logging/v1/kibana_types.go

```diff
@@ -0,0 +1,60 @@
+package v1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// KibanaSpec defines the desired state of Kibana
+// +k8s:openapi-gen=true
+type KibanaSpec struct {
+	Image           string                   `json:"image"`
+	ManagementState ManagementState          `json:"managementState"`
+	Resources       *v1.ResourceRequirements `json:"resources"`
+	NodeSelector    map[string]string        `json:"nodeSelector,omitempty"`
+	Tolerations     []v1.Toleration          `json:"tolerations,omitempty"`
+	Replicas        int32                    `json:"replicas"`
+	ProxySpec       `json:"proxy,omitempty"`
```

Comment:
this was confusing to look at at first. can we mirror what we have for `ManagementState` above and include the field name of `ProxySpec`?

Reply:
This was designed in the CLO, I just followed that design. You want `ProxySpec` field not to be embedded but added as a field like `ProxySpec ProxySpec`?

Reply:
if its how it was in CLO then we can leave it, i couldn't remember how it was defined there.